sm/plugins/tedge_apt_plugin/src/main.rs

```diff
@@ -48,6 +60,20 @@ impl InternalError {
     }
 }
 
+#[derive(Debug, Deserialize)]
+#[serde(rename_all = "lowercase")]
+enum UpdateAction {
+    Install,
+    Remove,
+}
```

Comment:
UpdateOperation makes more sense I feel.

Reply:
Yeah I had initially named it as `UpdateOperation` but then was worried if it would conflict with the whole Cumulocity `Operation` concept where software update, software list, device control etc are termed as supported "operations".